### ISSUE TITLE
Feature (server): add support for CORS headers to be set on the response

### ISSUE THREAD

**Issue by cipriancaba** [maoosi/prisma-appsync#55]
This would be useful when developing the api locally, that would start eg on localhost:4000 and you want to access it from a localhost:3000

**Comment by maoosi** [comment]
From `1.0.0-rc.4`, the local dev server will use GraphQL Yoga v3 (upgraded from v2). As part of it, I've added the ability to pass custom options to Yoga via `createServer({ yogaServerOptions })`.

So once it is released, you will be able to use CORS doing the following inside the `server.ts` file:

```ts
createServer({
  yogaServerOptions: {
    cors: {
      origin: 'http://localhost:4000',
      credentials: true,
      allowedHeaders: ['X-Custom-Header'],
      methods: ['POST']
    }
    /* ...other args */
  }
})
```

**Comment by maoosi** [comment]
Available in [1.0.0-rc.4](https://github.com/maoosi/prisma-appsync/releases/tag/1.0.0-rc.4).
